cilium/cmd/config.go

```diff
@@ -75,6 +75,8 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 
```

Comment:
Why part of the `cilium config` command? I think it should make more sense as part of the `cilium status` on the `K8s api server`